ci_framework/roles/ci_setup/tasks/packages.yml

```diff
@@ -8,3 +8,40 @@
   ansible.builtin.package:
     name: "{{ cifmw_ci_setup_packages }}"
     state: latest
+
+- name: Ensure openshift client install path is present
+  ansible.builtin.file:
+    path: "{{ cifmw_ci_setup_openshift_client_install_path }}"
+    state: directory
+    mode: 0755
+
+- name: Install openshift client
+  block:
+    - name: Gather version of openshift client if it exists
+      register: oc_version
+      environment:
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: "oc version -o yaml"
+
+    - name: Check if version of openshift client is supported
+      vars:
+        oc_version_data: "{{ oc_version.stdout | from_yaml }}"
+        oc_major_version: "{{ oc_version_data.releaseClientVersion | split('.') | first }}"
+      ansible.builtin.assert:
+        that:
+          - oc_major_version | int >= 4
+  rescue:
+    - name: Ensure openshift client install path is present
```

Comment:
This task is not needed as it was already created, if needed, [here](https://github.com/openstack-k8s-operators/ci-framework/pull/240/commits/456ecfcf8275ede2f507a82e1ef028c231373802#diff-bca68ddbe5009a351475b8cf4f9aed320d165c7be9403105689b3fad5832e96aR12)